src/orca/services/nextflowtower/client.py

```diff
@@ -0,0 +1,128 @@
+from typing import Any
+
+import requests
+from pydantic.dataclasses import dataclass
+
+
+@dataclass(kw_only=False)
+class NextflowTowerClient:
+    """Simple Python client for making requests to Nextflow Tower.
+
+    Attributes:
+        api_endpoint: API endpoint for a Nextflow Tower platform.
+        auth_token: An authentication token for the platform specified
+            by the ``api_endpoints`` value.
+    """
+
+    auth_token: str
+    api_endpoint: str
```

Comment:
In `sagetasks` we had a set list of API endpoints in a dictionary similar to the `SevenBridges` implementation in this repo. Is the reason to not include that here to allow for non-sage use cases?

Reply:
In the case of SevenBridges, there are no other possible options for `api_endpoint`. On the other hand, there are multiple deployments of Nextflow Tower. So, you are correct in saying that this decision is to allow for non-Sage use cases.